documentation/src/main/asciidoc/mapstruct-reference-guide.asciidoc

```diff
@@ -944,6 +944,8 @@ public interface CarMapper {
 
 * Between `java.time.ZonedDateTime`, `java.time.LocalDateTime`, `java.time.LocalDate`, `java.time.LocalTime` from Java 8 Date-Time package and `String`. A format string as understood by `java.text.SimpleDateFormat` can be specified via the `dateFormat` option (see above).
 
+* Between `java.time.Instant`, `java.time.Duration`, `java.time.Period` from Java 8 Date-Time package and `String`, as per the `parse` method in each class.
```

Comment:
as per .. is this correct english?.. Perhaps: "it uses the `parse` method in each class"

Reply:
I think you are right, changed it